src/features/multichain/hooks/__tests__/useCompatibleNetworks.test.ts

```diff
@@ -184,55 +183,7 @@ describe('useReplayableNetworks', () => {
         saltNonce: '0',
         setupData,
       }
-      const { result } = renderHook(() => useReplayableNetworks(creationData, []))
-      expect(result.current).toHaveLength(4)
-      expect(result.current.map((chain) => chain.chainId)).toEqual(['1', '10', '100', '480'])
-    }
-  })
-
-  it('should remove already deployed chains from result', () => {
```

Comment:
I moved this filtering to outside the hook because I needed to distinguish between
- there being no more chains left to deploy to (already deployed on all chains)
- there being no chains at all (unsupported safe creation version)